kuksa-client/docs/examples/async-grpc.md

```diff
@@ -18,6 +18,8 @@ This API makes the assumption you're interested in getting/setting/subscribing t
 
 Hence this is the go-to API if you're seeking minimum boilerplate and the most basic functionality.
 
+#### Unauthorized
```

Comment:
Would it possibly be better to have more elaborate headings, like

#### Connecting to a Databroker (without Authorization)


#### Connecting to a Databroker (with Authorization)